### PR TITLE
Jboss web

### PR DESCRIPTION
**Pull request by Emmelie83**
Changed the context root so that projektJakarta-1.0-SNAPSHOT is not needed in the path any more.

### CODE REVIEW

File: src/main/webapp/WEB-INF/jboss-web.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jboss-web xmlns="http://www.jboss.com/xml/ns/javaee"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="
+      http://www.jboss.com/xml/ns/javaee
+      http://www.jboss.org/j2ee/schema/jboss-web_5_1.xsd">
+  <context-root>/</context-root>
+</jboss-web>
```
Comment (Mattan41) [inline src/main/webapp/WEB-INF/jboss-web.xml:8]: shouldn't WEB-INF be under resources?

File: src/main/webapp/WEB-INF/web.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<web-app xmlns="https://jakarta.ee/xml/ns/jakartaee"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="https://jakarta.ee/xml/ns/jakartaee https://jakarta.ee/xml/ns/jakartaee/web-app_6_0.xsd"
+         version="6.0">
+  <error-page>
+    <error-code>404</error-code>
+    <location>/404.html</location>
+  </error-page>
+</web-app>
```
Comment (Mattan41) [inline src/main/webapp/WEB-INF/web.xml:10]: shouldn't WEB-INF be under resources?

**Review by Mattan41 — COMMENTED** [review]
shouldn't WEB-INF be under resources?

File: src/main/webapp/WEB-INF/jboss-web.xml (hunk 1 above)
Comment (Emmelie83) [reply src/main/webapp/WEB-INF/jboss-web.xml:8]: It didn't work when I put it there. But you can try it out. Maybe it works for you.

**Pull request merged**